src/app/content/components/Page/validateDOMContent.ts

```diff
@@ -25,6 +26,11 @@ const validateLinks = (rootEl: HTMLElement) => {
   if (config.UNLIMITED_CONTENT) {
     // tslint:disable:no-console
     console.warn(message);
+    Array.from(rootEl.querySelectorAll('a[href^="/"]')).forEach((element) => {
+      const href = assertNotNull(element.getAttribute('href'), 'href was null');
+      element.setAttribute('data-broken-href', href);
```

Comment:
What's the purpose of this attribute?

Reply:
I wanted to leave a note for a future developer to know that the attribute was removed intentionally rather than being missing in the content. Since this PR is temporary it is unlikely that we'll forget why there are a few links that are missing an href attribute; should it be removed?

Reply:
Maybe an open PR that reverses those changes, and explains why they were made, would be more noticeable?
I'm not really sure which is better so it's probably fine to keep it this way.